src/pageObject.js

```diff
@@ -1,16 +1,36 @@
 'use strict';
 
 class Header {
-
-};
+  get logo() {
+    return 'Conduit logo';
+  }
+}
 
 class PageObject {
-
-};
+  constructor(url) {
+    this.url = url;
+    this.header = new Header();
+  }
+  clickOnLogo() {
+    return 'Click on the ' + this.header.logo;
+  }
+}
 
 class ArticlePage extends PageObject {
-
-};
+  constructor(url, slug) {
+    super(url + slug);
```

Comment:
as for me, super(url) is enough
'slug' doesn't need to be passed to the parent constructor.

[checklist]: Constructor should modify original url by adding slug at the end